wolfcrypt/src/asn.c

```diff
@@ -218,6 +218,88 @@ extern int wc_InitRsaHw(RsaKey* key);
     #endif /* HAVE_SELFTEST */
 #endif
 
+#if !defined(WOLFSSL_NO_ASN_PRINT) && !defined(WOLFSSL_DEBUG_ASN_TEMPLATE)
```

Comment:
Why does `WOLFSSL_DEBUG_ASN_TEMPLATE` conflict? Did you mean ` || defined(WOLFSSL_DEBUG_ASN_TEMPLATE)`?

Reply:
Fixed.